hermes-console/static/js/console/owner/OwnerSelector.js

```diff
@@ -42,6 +43,22 @@ angular.module('hermes.owner')
         $scope.owners = function(searchString) {
             return ownerRepository.getOwners($scope.sourceSelectModel.name, searchString);
         };
+
+        $scope.placeholder = function () {
```

Comment:
This name may be too generic for global scope, maybe something like `ownerPlaceholder`?